lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -170,12 +172,24 @@ def list_attachment_filenames(attachments)
       ###
 
       def translate_banking_info
+        monitor = DisabilityCompensation::Loggers::Monitor.new
+
         populated = input_form['bankName'].present? && input_form['bankAccountType'].present? &&
                     input_form['bankAccountNumber'].present? && input_form['bankRoutingNumber'].present?
         # If banking data is not included, then it has not changed and will be retrieved from Lighthouse
         if !populated || redacted(input_form['bankAccountNumber'], input_form['bankRoutingNumber'])
-          get_banking_info
+          monitor.track_526_submission_without_banking_info(@user.uuid)
```

Comment:
There is a lot of code traversed through here, do you know, does this code wrap its own error if it has any, or like... if this method throws an error, it would tank a submission, which seems like it shouldnt since it is just logging stuff... 

Or like could/should this be behind the flipper too? IE

```ruby
if Flipper.enabled?(:disability_526_block_banking_info_retrieval)
  monitor.track_526_submission_without_banking_info(@user.uuid) 
  {} 
else
  get_banking_info
end
```

Just to reduce risk on release?